tests/e2e/performance/test_pvc_creation_deletion_performance.py

```diff
@@ -150,12 +193,22 @@ def test_pvc_creation_deletion_measurement_performance(
         )
         self.full_results.add_key("pvc_size", pvc_size)
         num_of_samples = 5
-        accepted_creation_time = 1
-        accepted_deletion_time = 2 if self.interface == constants.CEPHFILESYSTEM else 1
+        accepted_creation_time = (
+            3600 if self.interface == constants.CEPHBLOCKPOOL_THICK else 1
+        )
+
+        # accepted deletion time for RBD is 1 sec, for CephFS is 2 secs and for RBD Thick is 5 secs
+        if self.interface == constants.CEPHFILESYSTEM:
+            accepted_deletion_time = 2
+        elif self.interface == constants.CEPHBLOCKPOOL:
+            accepted_deletion_time = 1
+        else:
+            accepted_deletion_time = 5
+
         self.full_results.add_key("samples", num_of_samples)
 
-        accepted_creation_deviation_percent = 50
-        accepted_deletion_deviation_percent = 50
+        accepted_creation_deviation_percent = 80
```

Comment:
We've had several conversations around the deviation. creation time for rbd and cephfs can simply be verified to be less than 1 second and we can skip the calculation of deviation for cephfs & rbd. WDYT?

Reply:
I think that in case creation /deletion time is less then 1 second - calculating and checking standard deviation is not relevant. 
And if those times are bigger than 1 second - we'll fail the test ( for CephFS and for RBD). 

As for rbd thick - I think that Standard Deviation calculations are irrelevant as well. Actual results show that the deviation in this case is often more than 50%. 

So should I just remove all the calculations of standard deviation from this test ?